src/collection/collection_test.go

```diff
@@ -17,10 +17,12 @@ func Test_buildSchemaListForDatabase(t *testing.T) {
 		"index_name",
 	}).AddRow("schema1", "table1", "index1")
 
-	mock.ExpectQuery(".*").WillReturnRows(instanceRows)
+	mock.ExpectQuery(dbSchemaQuery).WillReturnRows(instanceRows)
+	mock.ExpectClose()
 
-	schemaList, err := buildSchemaListForDatabase("database1", testConnection)
+	schemaList, err := buildSchemaListForDatabase(testConnection)
 	assert.Nil(t, err)
+	testConnection.Close()
```

Comment:
You can ignore this, but normally I tend to put the connection closing as a defer just after the connection creation.

Reply:
if i do this the test will fail since I 'm executing `assert.NoError(t, mock.ExpectationsWereMet())` before this func ends.

Reply:
oks